tests/system/python/test_e2e_expr_pi.py

```diff
@@ -0,0 +1,116 @@
+# -*- coding: utf-8 -*-
+
+# FOGLAMP_BEGIN
+# See: http://foglamp.readthedocs.io/
+# FOGLAMP_END
+
+""" Test end to end flow with,
+
+        Expression south plugin
+        Metadata filter plugin
+        PI Server (C) plugin
+"""
+
+
+import http.client
+import json
+import time
+import pytest
+
+
+__author__ = "Praveen Garg"
+__copyright__ = "Copyright (c) 2019 Dianomic Systems"
+__license__ = "Apache 2.0"
+__version__ = "${VERSION}"
+
+
+SOUTH_PLUGIN = "Expression"
+SOUTH_PLUGIN_LANGUAGE = "C"
+
+SVC_NAME = "Expr #1"
+ASSET_NAME = "Expression"
+
+
+class Test_E2E_Expr_Pi:
+
+    @pytest.fixture
+    def start_south_north(self, reset_and_start_foglamp, start_south, remove_data_file,
```

Comment:
Add `filter_branch, filter_plugin, filter_name, `

Reply:
removing filter_plugin as it will be test speicific for assertions. Fixture will allow to pass arg though so that any test can use it. argparser allows branch and name for test with diff name and branches for used plugin

Reply:
> filter_plugin Fixture will allow to pass arg though so that any test can use it

If this test case only compatible with `metadata` then it is fine